bgpd/bgp_nb_config.c

```diff
@@ -173,6 +173,16 @@ int bgp_router_destroy(struct nb_cb_destroy_args *args)
 	case NB_EV_APPLY:
 		bgp = nb_running_unset_entry(args->dnode);
 
+		/* make sure we withdraw any exported routes */
+		if (CHECK_FLAG(bgp->af_flags[AFI_IP][SAFI_UNICAST],
```

Comment:
If you look in vpn_leak_prechange it does this if check, why do we neeed to do it again here?

Reply:
> If you look in vpn_leak_prechange it does this if check, why do we neeed to do it again here?

I shall reemove the check and simply call the 'prechange' function for each AF - incoming